libs/ui/src/contest_tally.tsx

```diff
@@ -319,9 +320,7 @@ export function ContestTally({
         return (
           <Contest key={`div-${contest.id}`} dim={!talliesRelevant}>
             <Prose maxWidth={false} data-testid={`results-table-${contest.id}`}>
-              {contest.section !== contest.title && (
-                <Text small>{contest.section}</Text>
-              )}
+              <Text small>{getContestDistrictName(election, contest)}</Text>
               <h3>
```

Comment:
This is probably the biggest UX change in the whole PR. Now for all definitions, a district name subtitle will appear above every contest tally in a report. If we want to hide that, we should probably have a better pattern than checking if `contest.section === contest.title`.